packages/connector-alipay-native/README.md

```diff
@@ -2,25 +2,50 @@
 
 Alipay Native connector works closely with Logto SDK on mobile platforms. It takes advantage of Alipay's OAuth 2.0 authentication workflow and enables Alipay users to sign in to other Apps using public Alipay user profiles without going through a troublesome register process.
 
+支付宝原生连接器与 Logto 所提供的原生平台上的 SDK 紧密搭配使用。 它利用支付宝所提供的 OAuth 2.0 身份认证服务，使支付宝用户无需繁琐的注册流程，即可直接用其在支付宝上公开的身份信息登录其他应用。
```

Comment:
just wonder: should we add Chinese description here?

Reply:
Yes, just align with WeChat native connector readme.